extensions/ql-vscode/src/view/result-tables.tsx

```diff
@@ -112,20 +114,22 @@ export class ResultTables
 
   /**
    * Holds if we actually should show pagination interface right now. This is
-   * still false for the time being when we're viewing alerts.
+   * true as long as the result sets we're given are marked to permit it.
```

Comment:
should probably just get rid of this function since it duplicates `paginationAllowed` without adding anything new. And `paginationAllowed` will be removed when we remove the pagination flag, right?

Reply:
Sure, could just inline that now.